packages/atlas/src/config/channelFilter.ts

```diff
@@ -0,0 +1,7 @@
+import { ChannelWhereInput } from '@/api/queries'
+
+export const channelFilter: ChannelWhereInput = {
+  isCensored_eq: false,
+  isPublic_eq: true,
+  NOT: [{ id_in: [] }, { avatarPhoto: { id_in: [] } }, { coverPhoto: { id_in: [] } }],
+}
```

Comment:
I did some testing and is this actually working? For example I added to the list channel id "2"
- I still see videos from this channel in `popular on joystream` and in `top 10 this week`. 
- I still can go to the channel with id "2" and see all the content.

I think we will have to also use these channel filters partly in the video filter to make this right - (`videos where  channel id_in`). It's also very possible that Orion doesn't apply these filters and we will need to update Orion in that regard, but not super sure about this.

Reply:
@drillprop I cannot confirm first point. For the filter below videos `13` and `2` are not showing
```ts
export const videoFilter: VideoWhereInput = {
  isCensored_eq: false,
  isPublic_eq: true,
  thumbnailPhoto: {
    isAccepted_eq: true,
  },
  media: {
    isAccepted_eq: true,
  },
  NOT: [{ id_in: ['13', '2'] }, { thumbnailPhoto: { id_in: [] } }, { media: { id_in: [] } }],
}

```

Reply:
but, you're referring to the video filter. I was referring to the channel filter. If you block the channel, you shouldn't see videos from this channel on the homepage. Am I right?

Reply:
@drillprop  I created a fix, unfortunately due to nesting problem we can only filter by channel id